src/conversions/fixed.rs

```diff
@@ -0,0 +1,1178 @@
+#![cfg(feature = "fixed")]
+//! Conversions to and from [fixed](https://docs.rs/fixed)'s type.
+//!
+//! This is useful for converting Python's decimal.Decimal into and from a native Rust type (fixed).
+//!
+//! # Setup
+//!
+//! To use this feature, add to your **`Cargo.toml`**:
+//!
+//! ```toml
+//! [dependencies]
+#![doc = concat!("pyo3 = { version = \"", env!("CARGO_PKG_VERSION"),  "\", features = [\"fixed\"] }")]
+//! fixed = "1.23.1"
+//! ```
+
+use crate::exceptions::PyValueError;
+use crate::once_cell::GILOnceCell;
+use crate::types::PyType;
+use crate::{intern, FromPyObject, IntoPy, Py, PyAny, PyObject, PyResult, Python, ToPyObject};
+use fixed::types::*;
+use std::str::FromStr;
+
+static DECIMAL_CLS: GILOnceCell<Py<PyType>> = GILOnceCell::new();
```

Comment:
We cache this three times now, don't we? I think it might be useful to centralize the `get_decimal_cls` helper function somewhere.